tests/test_plotting.py

```diff
@@ -68,3 +74,25 @@ def test_gain_chart(classification_dataset):
 
     # Check that the figure is a matplotlib Figure
     assert isinstance(fig, Figure), "plot_gain_chart did not return a matplotlib Figure"
+
+
+def test_plot_regression_predicted_vs_actual(regression_dataset):
```

Comment:
Nitpicky but you could add a test of the `alpha` argument as well, or as part of the same test with another assert